whisper_online.py

```diff
@@ -611,14 +622,18 @@ def asr_factory(args, logfile=sys.stderr):
     logfile = sys.stderr
 
     if args.offline and args.comp_unaware:
-        print("No or one option from --offline and --comp_unaware are available, not both. Exiting.",file=logfile)
+        logger.error("No or one option from --offline and --comp_unaware are available, not both. Exiting.")
```

Comment:
I think command line errors go to stderr explicitly. If you set the log level too high the command may confusingly fail without error message. (I think the `file` argument in the original was also a mistake.)